server.js

```diff
@@ -1369,42 +1366,26 @@ module.exports.initExpress = function () {
   );
   app.use(
     '/pl/course_instance/:course_instance_id/instance_question/:instance_question_id/clientFilesQuestion',
-    [
-      require('./middlewares/selectAndAuthzInstanceQuestion'),
-      require('./middlewares/studentAssessmentAccess'),
-      require('./pages/clientFilesQuestion/clientFilesQuestion'),
-    ]
+    [require('./pages/clientFilesQuestion/clientFilesQuestion')]
```

Comment:
We can get rid of the `[]` here if we want (and similarly elsewhere).
```suggestion
    require('./pages/clientFilesQuestion/clientFilesQuestion')
```

Reply:
Done in 2aaec9c.

Reply:
Not that it really matters, but it looks like you missed the ones for `legacyQuestionFile` and `legacyQuestionText`.

Reply:
Whoops, indeed - addressed in 5e59a34.